### PR TITLE
add testBookingConfirmation code

### PR DESCRIPTION
**Pull request by AlexDiZhu**

### CODE REVIEW

**Comment by AlexDiZhu** [comment]
add testBookingConfirmation

**Comment by ezeutno** [comment]
Is this finished yet?

File: App/test/app/TestBookingConfirmation.java [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+                Date dateout = Date.valueOf(read.nextLine());
+                System.out.println("SQL date in " + datein);
+                System.out.println("SQL date out " + dateout);
+                System.out.println("\noutput of checkAvailability() = " + testSubject.checkAvailability(HID, datein, dateout));
+                i++;
+            }
+        } catch (FileNotFoundException ex) {
+            Logger.getLogger(TestBookingConfirmation.class.getName()).log(Level.SEVERE, null, ex);
+        }
+    }
```
Comment (ezeutno) [inline App/test/app/TestBookingConfirmation.java:106]: This is @kyemurf Code ❓

**Comment by AlexDiZhu** [comment]
add testconfirmation

**Pull request closed without merging**